tests/catch/CMakeLists.txt

```diff
@@ -116,6 +116,17 @@ if(NOT DEFINED OFFLOAD_ARCH_STR AND EXISTS "${ROCM_PATH}/bin/rocm_agent_enumerat
     endif()
 endif()
 
+# In windows get the gpu targets list from cmd line
+if (HIP_PLATFORM STREQUAL "amd" AND NOT UNIX)
```

Comment:
Why limit this to only windows? We could make use of this infrastructure on linux as well.

Reply:
removed NOT UNIX check. Using the command line GPU_ARCH_LIST for all OS